docs/clone-sst.md

```diff
@@ -0,0 +1,190 @@
+
+For Percona XtraDB Cluster, any variables related to the SST mechanism, such as `wsrep_sst_allowed_methods` and `wsrep_sst_method`, must be defined before the  server startup to ensure proper synchronization.
+
+## Enable SSL for Clone SST
+
+To enable SSL for the Clone SST process, place the SSL certificates in a directory other than the data directory, as the clone process modifies this directory.
+
+We recommend explicitly setting the SSL certificates in the `my.cnf` file as follows:
+
+```ini
+[client]
+ssl-ca = /<path>/ca.pem
+ssl-cert = /<path>/client-cert.pem
+ssl-key = /<path>/client-key.pem
+
+[mysqld]
+ssl-ca = /<path>/ca.pem
+ssl-cert = /<path>/server-cert.pem
+ssl-key = /<path>/server-key.pem
+```
+
+Alternatively, you can configure the following SSL settings specifically for the Clone SST process on the Joiner:
+
+```ini
+[mysqld]
+clone_ssl_ca = /path/to/ca.pem
+clone_ssl_cert = /path/to/client-cert.pem
+clone_ssl_key = /path/to/client-key.pem
+```
+
+Ensure the `<path>` used is not the data directory to avoid conflicts during the Clone SST process.
+
+## Variables
+
+### Common variables
+
+State Snapshot Transfer (SST) in Galera Cluster relies on specific variables that control its configuration and behavior. YOu must set these variables appropriately to ensure seamless synchronization between nodes during the SST process. The following lists of the most commonly used variables and their purposes.
+
+* **`sst-idle-timeout`**:  
+  Sets the maximum time (in seconds) the SST process can remain idle before being considered failed.  You must define this variable in the `[sst]` section of the `my.cnf` file.
+  [Learn more](wsrep-system-index.md#sst-idle-timeout)
+
+* **`wsrep_sst_donor`**:  
+  Defines the preferred donor node for SST. If not specified, the cluster automatically selects a donor.  
+  [Learn more](wsrep-system-index.md#wsrep_sst_donor)
+
+
+* **`wsrep_sst_method`**:  
+  Specifies the method or script used for the State Snapshot Transfer (SST) process. Only one value can be selected.  
+  [Learn more](wsrep-system-index.md#wsrep_sst_method)
+
+* **`wsrep_sst_receive_address`**:  
+  Specifies the IP address and port on the Joiner node to receive SST data.  
+  [Learn more](wsrep-system-index.md#wsrep_sst_receive_address)
+
+### Timeout variables
+
+During the Clone SST process, there are three key moments when the Joiner or Donor must wait for the other to complete a specific action. These moments are governed by the following configurable timeout variables:
+
+- **`joiner-timeout-wait-donor-message`**: Determines how long (in seconds) the Joiner waits for the Donor to respond, indicating whether the action to perform is SST or IST. The default value is 60 seconds, which is usually sufficient. If the timeout is reached, the process aborts.
```

Comment:
likewise. The convention is to use underscores in my.cnf (yes, it will work with dashes, but I think it is better to be consistent in the documentation)
@venkatesh-prasad-v what is your opinion?

Reply:
Yes. It works with both underscores and dashes. As you said, let's have it consistent everywhere.